projects/ion/src/core/pipes/currency.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import { CurrencyPipe } from '@angular/common';
+import { PipeStrategy } from './pipe-strategy';
+
+export class CurrencyPipeStrategy implements PipeStrategy {
+  transform(value: number): string {
+    const locale = 'en-US';
```

Comment:
In order to change this locale to 'pt-BR', you'll need to add these three lines on `ion.module.ts`, right before NgModule decorator:
```
import { registerLocaleData } from '@angular/common';
import localePT from '@angular/common/locales/pt';

registerLocaleData(localePT);
```
Same on datePipe. This will prevent us to have english patterns being setted, like this one:
![image](https://github.com/Brisanet/ion/assets/138066684/68332d13-4b00-4d2c-bc5a-cf2a9ed5a315)